src/components/ui/VertialDelimiter.tsx

```diff
@@ -0,0 +1,11 @@
+import classnames, { backgroundColor, height, width } from 'classnames/tailwind'
+
+const vertialDelimiter = classnames(
+  height('h-px'),
+  width('w-full'),
+  backgroundColor('bg-divider')
+)
+
+export default function () {
+  return <div className={vertialDelimiter}></div>
```

Comment:
We already have one vertical delimiter component (`LastDelimiter` or `Delimiter`, not sure), let's use that one + let's close the div tag like this: `<div />`